### PR TITLE
BETA: Register charlesbird.is-a.dev

### PR DESCRIPTION
**Pull request by UntoldGam**
Added `charlesbird.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/logging-server.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "UntoldGam",
+        "email": "charlesobird+github@outlook.com"
+    },
+    "record": {
+        "A": 130.162.166.23
```
Comment (CuteDog5695) [inline domains/logging-server.json:7]: IP needs to be in a array

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/logging-server.json (hunk 1 above)
Comment (Omxpro) [reply domains/logging-server.json:7]: so quick

Comment (UntoldGam) [reply domains/logging-server.json:7]: I didn't mean for that file to be sent, the pr was done via the website. Is it alright to close this and make a new pr?

Comment (UntoldGam) [reply domains/logging-server.json:7]: I don't need that file, it's the charlesbird.json one that i need

Comment (Omxpro) [reply domains/logging-server.json:7]: > I don't need that file, it's the charlesbird.json one that i need

delete it then

**Review by UntoldGam — COMMENTED** [review]
this is the correct file that I want uploading

**Review by Omxpro — CHANGES_REQUESTED** [review]
CNAME cannot contain https://

**Comment by Omxpro** [comment]
and cname must be githubuser.github.io

**Review by UntoldGam — COMMENTED** [review]
removed https:// and the / at the end

**Review by Omxpro — APPROVED** [review]
looks alr but website in incomplete

**Comment by UntoldGam** [comment]
I haven't completed it yet, should I wait until it's completed to set this up?

**Comment by Omxpro** [comment]
> I haven't completed it yet, should I wait until it's completed to set this up?

i have submitted the pr for review. you can continue doing what you want. it will be merged shortly

File: domains/charlesbird.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  {
```
Comment (CuteDog5695) [inline domains/charlesbird.json:2]: remove the { here, its making the file invalid

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by UntoldGam — COMMENTED** [review]
removed the { at line 2

**Review by iostpa — APPROVED** [review]
now its fine

**Pull request merged**